stablehlo/dialect/VersionTest.cpp

```diff
@@ -0,0 +1,17 @@
+#include "gtest/gtest.h"
+#include "mlir/Bytecode/Encoding.h"
+#include "stablehlo/dialect/Version.h"
+
+namespace mlir {
+namespace stablehlo {
+namespace {
+
+TEST(VersionTest, CurrentBytecodeVersionIsLatestUpstream) {
```

Comment:
Looks like OpenXLA tends to place unit tests next to files. No flags here. Alternate would be in the `unittests` dir.